### PR TITLE
Found a way to handle navbar collapse on small devices

### PR DESCRIPTION
**Pull request by zhryn**

### CODE REVIEW

File: src/Collapse.svelte [hunk 1]
```diff
@@ -13,23 +14,44 @@
...
-    _wasOpen = isOpen;
-    isOpen = true;
-  } else if (windowWidth < 768) {
-    isOpen = _wasOpen;
+  const minWidth = {};
+  minWidth['xs'] = 0;
+  minWidth['sm'] = 576;
+  minWidth['md'] = 768;
+  minWidth['lg'] = 992;
+  minWidth['xl'] = 1200;
```
Comment (bestguy) [inline src/Collapse.svelte]: This is a good improvement over current behavior, though one comment is can you think of a way that we can avoid hardcoding these breakpoints?

The concern is that these are actually sass variables and could be different in some Bootstrap themes:

https://github.com/twbs/bootstrap/blob/master/scss/_variables.scss#L186

**Comment by bestguy** [comment]
Thanks, will merge fix and deal with my comment in a separate PR

File: src/Collapse.svelte (hunk 1 above)
Comment (zhryn) [reply src/Collapse.svelte]: You might create a div element inside body tag like this to find which viewport used (xs, sm, md, lg, or xl)
```
<div id="sizer">
    <div class="d-block d-sm-none d-md-none d-lg-none d-xl-none" data-size="xs"></div>
    <div class="d-none d-sm-block d-md-none d-lg-none d-xl-none" data-size="sm"></div>
    <div class="d-none d-sm-none d-md-block d-lg-none d-xl-none" data-size="md"></div>
    <div class="d-none d-sm-none d-md-none d-lg-block d-xl-none" data-size="lg"></div>
    <div class="d-none d-sm-none d-md-none d-lg-none d-xl-block" data-size="xl"></div>
</div>
```

and then
```
const breakpoint = document.querySelector("#sizer").querySelectorAll("div:visible").data("size")
```

Comment (bestguy) [reply src/Collapse.svelte]: Thanks for the idea 👍 will take a look

**Pull request merged**